okapi-core/src/test/java/org/folio/okapi/util/ScheduleTest.java

```diff
@@ -0,0 +1,208 @@
+package org.folio.okapi.util;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import java.time.Duration;
+import java.time.LocalDateTime;
+import java.util.LinkedList;
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class ScheduleTest {
+
+  @Test
+  void testParseComp1() {
+    List<Integer> l = new LinkedList<>();
+    Schedule.parseComp(l, "32", 0, 59);
+    assertThat(l).containsExactlyInAnyOrder(32);
+  }
+
+  @Test
+  void testParseComp2() {
+    List<Integer> l = new LinkedList<>();
+    Schedule.parseComp(l, "3,7,59", 0, 59);
+    assertThat(l).containsExactlyInAnyOrder(3, 7, 59);
+  }
+
+  @Test
+  void testParseComp3() {
+    List<Integer> l = new LinkedList<>();
+    Schedule.parseComp(l, "*", 2, 5);
+    assertThat(l).containsExactlyInAnyOrder(2, 3, 4, 5);
+  }
+
+  @Test
+  void testParseComp4() {
+    List<Integer> l = new LinkedList<>();
+    Schedule.parseComp(l, "*/15", 4, 63);
+    assertThat(l).containsExactlyInAnyOrder(4, 19, 34, 49);
+  }
```

Comment:
testParseComp1 - testParseComp4 should be merged:

```
  static Stream<Arguments> testParseComp() {
    return Stream.of(
        Arguments.of("32",     0, 59, new Integer []{ 32 }),
        Arguments.of("3,7,59", 0, 59, new Integer []{ 3, 7, 59 }),
        Arguments.of("*",      2, 5,  new Integer []{ 2, 3, 4, 5 }),
        Arguments.of("*/15",   4, 63, new Integer []{ 4, 19, 34, 49 })
        );
  }

  @ParameterizedTest
  @MethodSource
  void testParseComp(String spec, int min, int max, Integer [] expected) {
    List<Integer> l = new LinkedList<>();
    Schedule.parseComp(l, spec, min, max);
    assertThat(l).containsExactlyInAnyOrder(expected);
  }
```

Similar for testParseCompFail and testSchedule*.